### PR TITLE
MINOR: Refresh of the docs

### PR DESCRIPTION
**Pull request by mimaison**
- Remove line numbers for non-code formatted blocks
- Use consistent prompt $ for all bash commands
- Remove outdated zookeeper content
- Cleanup of HTML formatting

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Review by showuon — COMMENTED** [review]
LGTM! But it might be better if you could run this doc change locally and have screenshot to make sure it works as expected, especially the quickstart section. Thanks.

**Comment by mimaison** [comment]
It's not easy since this includes changes all over the website. Like for code changes, reviewers should test this change locally.

**Review by showuon — APPROVED** [review]
LGTM! Thanks.

**Comment by mimaison** [comment]
If it's a pain to review, I can split the PR.

**Comment by showuon** [comment]
> If it's a pain to review, I can split the PR.

It's fine. It looks good. Thanks.

File: docs/streams/developer-guide/app-reset-tool.html [hunk 1]
```diff
@@ -78,9 +78,9 @@
             <h2>Step 1: Run the application reset tool<a class="headerlink" href="#step-1-run-the-application-reset-tool" title="Permalink to this headline"></a></h2>
             <p>Invoke the application reset tool from the command line</p>
             <p>Warning! This tool makes irreversible changes to your application. It is strongly recommended that you run this once with <code class="docutils literal"><span class="pre">--dry-run</span></code> to preview your changes before making them.</p>
-            <pre class="line-numbers"><code class="language-bash">&lt;path-to-kafka&gt;/bin/kafka-streams-application-reset</code></pre>
+            <pre<code class="language-bash">$ bin/kafka-streams-application-reset</code></pre>
```
Comment (chia7712) [inline docs/streams/developer-guide/app-reset-tool.html]: `<pre<code` -> `<pre><code`

File: docs/implementation.html [hunk 2]
```diff
@@ -75,13 +75,13 @@ <h4 class="anchor-heading"><a id="recordbatch" class="anchor-link"></a><a href="
...
     <p> The key of a control record conforms to the following schema: </p>
-    <pre class="line-numbers"><code class="language-text">version: int16 (current version is 0)
+    <pre><code class="language-text">version: int16 (current version is 0)
 type: int16 (0 indicates an abort marker, 1 indicates a commit)</code></pre>
     <p>The schema for the value of a control record is dependent on the type. The value is opaque to clients.</p>
 
     <h4 class="anchor-heading"><a id="record" class="anchor-link"></a><a href="#record">5.3.2 Record</a></h4>
 	<p>Record level headers were introduced in Kafka 0.11.0. The on-disk format of a record with Headers is delineated below. </p>
-	<pre class="line-numbers"><code class="language-text">length: varint
+	<pre"><code class="language-text">length: varint
```
Comment (chia7712) [inline docs/implementation.html]: `<pre">` -> `<pre>`

**Review by chia7712 — COMMENTED** [review]
@mimaison nice patch. two comments are left

File: docs/streams/developer-guide/app-reset-tool.html (hunk 1 above)
Comment (mimaison) [reply docs/streams/developer-guide/app-reset-tool.html]: Fixed

File: docs/implementation.html (hunk 2 above)
Comment (mimaison) [reply docs/implementation.html]: Fixed

**Review by chia7712 — APPROVED** [review]
LGTM

**Comment by mimaison** [comment]
Since all changes are to `.html` files, I'll not wait for tests to complete. Merging to trunk.

**Comment by C0urante** [comment]
@mimaison would it be possible to backport this to 3.8, since it contains a syntax fix for the Connect quickstart? I've opened https://github.com/apache/kafka-site/pull/615 to port that specific change over to our docs site, but want to make sure it doesn't get overwritten when we publish 3.8.1.

**Comment by mimaison** [comment]
Yeah ideally we'd like want to sync all these to the website before 3.9. Should we wait for 3.8.0 to be out before backporting to 3.8?

**Comment by C0urante** [comment]
Yeah, that should be fine, thanks!

**Comment by mimaison** [comment]
Backported to 3.8: https://github.com/apache/kafka/commit/57c50e3b2ad7e3a0dc54715816103d59a447480a

**Comment by jlprat** [comment]
Shall we mimic this PR in `kafka-site`?

**Comment by mimaison** [comment]
Yes, I'll open a PR for kafka-site this afternoon.

**Comment by jlprat** [comment]
Ping me and I'll review it

**Pull request merged**